src/torchpme/calculators/calculator.py

```diff
@@ -158,6 +160,7 @@ def forward(
             positions=positions,
             neighbor_indices=neighbor_indices,
             neighbor_distances=neighbor_distances,
+            smearing=self.potential.smearing,
```

Comment:
Why do you have to pass the smearing here. Since it is an attribute of the calls you can just use `self.potential.smearing` inside of `_validate_compute_parameters`.

Reply:
Isn't `_validate_compute_parameters` a static method? I tried to add the zero cell check by introducing the fewest number of modifications, so I decided to keep it a static method and just pass the smearing to it